.teamcity/_self/projects/WebApp.kt

```diff
@@ -760,6 +760,7 @@ object RunCalypsoPlaywrightE2eTests : BuildType({
 				export LIVEBRANCHES=true
 				export NODE_CONFIG_ENV=test
 				export PLAYWRIGHT_BROWSERS_PATH=0
+				export FRAMEWORK=playwright
```

Comment:
IMO, the downsides of using an env var are:
- `FRAMEWORK` is too generic. I may have an env var already called `FRAMEWORK` in my laptop that will affect the process when running tests locally
- I may have a `FRAMEWORK=playwright` that I set months ago and forgot about it, which will cause problems if I try to run selenium tests locally.
- I need to specify I want playwright in two separate places and ensure they are in sync (eg: `FRAMEWORK=selenium yarn magellan --config=magellan-playwright.json` is mixing selenium and playwright).

I think it will be simpler if, instead of doing this, we pass `--mocha-args="--config .mocharc-playwright.yml --reporter mocha-teamcity-reporter"` in line 809. And then have two config files, `.mocharc.yml` and `.mocharc-playwright.yml`. Even if there is some duplication between those two files, that will be gone once we migrate all tests (or we move away from Mocha)

Reply:
I initially had it implemented as such, but I chose to do the unified mocharc.js file with conditional logic because:
- maintain existing workflow for users when they run `./run.sh` or `mocha specs/whatever`
- broke CircleCI runs, requiring me to add  `MOCHA_ARGS+='--config <file>` in one of the flags.

Reply:
> I need to specify I want playwright in two separate places and ensure they are in sync (eg: FRAMEWORK=selenium yarn magellan --config=magellan-playwright.json is mixing selenium and playwright).

This would be an intentional mix-up but is a real concern that I did not think about.

Reply:
Since my most recent push broke all tests, I will explore reverting to having two sets of mocha config files. It would make things more explicit and intentional, both good qualities to have for a test suite.